docs/modules/demos/pages/argo-cd-git-ops.adoc

```diff
@@ -0,0 +1,317 @@
+= argo-cd-git-ops
+:description: Deploy Stackable operators and products with ArgoCD. Sync manifests and secrets from Git.
+
+:k8s-cpu: https://kubernetes.io/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/#cpu
+:argo-cd: https://argoproj.github.io/cd/
+:argo-cd-application: https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#applications
+:argo-cd-applicationset: https://argo-cd.readthedocs.io/en/stable/user-guide/application-set/
+:sealed-secrets: https://github.com/bitnami-labs/sealed-secrets
+:stackable-demo-repository: https://github.com/stackabletech/demos/
+:airflow: https://airflow.apache.org/
+
+This demo shows how to utilize GitOps and Infrastructure as Code (IaC) with Stackable and {argo-cd}[ArgoCD]. 
+All products and manifests are synced and deployed via ArgoCD (except ArgoCD itself, which is bootstrapped via `stackablectl`).
+
+The key points to show are:
+
+* GitOps: How to deploy changes from a Git repository to my cluster?
+* Secrets: How to safely store and deploy credentials, access keys from a Git repository?
+* Interaction with Stackable products (e.g. Airflow and DAGs via gitsync)
+
+ArgoCD and other deployed products and dependencies are illustrated in the following diagram: 
+
+image::argo-cd-git-ops/architecture-overview.drawio.svg[]
+
+Install this demo on an existing Kubernetes cluster:
+
+TODO: We need a pointer here to the last paragraph (forking repo) if people want to "interact" and not just install the demo (install command requires extra parameters)
+
+[source,console]
+----
+$ stackablectl demo install argo-cd-git-ops --namespace argo-cd
+----
+
+WARNING: This demo should not be run alongside other demos.
+
+NOTE: ArgoCD will be deployed in the `argo-cd` namespace by `stackablectl`. ArgoCD itself will create other namespaces for the deployed products.
+
+[#system-requirements]
+== System requirements
+
+To run this demo, your system needs at least:
+
+* 15 {k8s-cpu}[cpu units] (core/hyperthread)
+* 15 GiB memory
+* 5 GiB disk storage
+
+== Overview
+
+This demo consists of multiple parts:
+
+* Bootstrapping via `stackablectl`:
+** Install {argo-cd}[Argo CD] via `stackablectl`
+* Deploy components via ArgoCD:
+** Install a {sealed-secrets}[Sealed Secrets] controller to handle sensitive data like credentials or secret keys
+** Install all Stackable operators using an `ApplicationSet`
+** Install requirements like Minio and Postgres as `Application`
+** Deploy Stackable Airflow manifests into their respective `Projects`
+* Inspect ArgoCD UI
+* Inspect Airflow and start DAGs
+** Checkout Airflow UI and synced DAGs (the DAGs are synced via gitsync, not ArgoCD)
```

Comment:
I think this (having two ways to update/sync elements of a cluster, either the cluster itself or the workflows) is a major feature of the demo and could maybe be given more prominence!

Reply:
Good point, i try to reformulate.

Reply:
Thanks!